locales/en-US/editor.properties

```diff
@@ -140,6 +140,19 @@ DND_SUCCESS_UNTAR=Successfully untarred <b>{0}</b>.
 # Image Viewer
 IMAGE_DIMENSIONS={0} (width) &times; {1} (height) pixels
 
+# Image Filters
+
+IMAGE_FILTER=Add Filter
```

Comment:
`IMAGE_FILTERS_TITLE` would make more sense to me, but it's not really important.